exonum/src/encoding/serialize/json.rs

```diff
@@ -316,6 +318,7 @@ impl<'a> ExonumJson for &'a [u8] {
     }
 }
 
+#[cfg_attr(feature = "cargo-clippy", allow(use_self))]
```

Comment:
Please here and below (if there are any more cases) either resolve the problem or document why `allow(use_self)` must be used.